lib/simple-dialog.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+/** @jsx etch.dom */
+
+import {CompositeDisposable, TextEditor} from 'atom'
+import etch from 'etch'
+import EtchComponent from './etch-component'
+
+type Props = {
+  prompt: string,
+  initialValue?: string,
+  onConfirm: (string) => void,
+  onCancel?: () => void
+}
+
+export default class SimpleDialog extends EtchComponent {
```

Comment:
Maybe we want to change this to actually have two components: `Dialog`, and `TextEditorDialog`

The `Dialog` component just displays any kind of children (passed in via the constructor) inside the `go-plus-dialog` div (inside an atom modal of course)

The `TextInputDialog` just uses the `Dialog` and passes the div with the prompt and the TextEditor as children to `Dialog`.

`Dialog` then could for example also be used for the `tags` dialog.

This is just a suggestion. I haven't thought it through yet. It could also be done later once we find the time and we need it 😉

Reply:
Seems overkill for now.  In fact I really didn't even need to make simple dialog as part of this PR.  When I first started out I thought I was going to need to make my own view for the drop down but then remembered `atom-select-list`.